File: Selenium/Activities/Activity4.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:
    driver.get("https://v1.training-support.net/selenium/target-practice")
    print("title of web pageis : ", driver.title)

    thirdHeading = driver.find_element(By.XPATH, "//h3[@id='third-header']")
    print("Third heading is : ", thirdHeading.text)

    fifthHeading = driver.find_element(By.XPATH, "//h5")
    print("Fifth heading colour is : ", fifthHeading.value_of_css_property("color"))

    violet = driver.find_element(By.CLASS_NAME, "violet")
    print("Violet button class is : ", violet.get_attribute("class"))

    grey = driver.find_element(By.CLASS_NAME, "grey")
    print("Text in grey button is: ", grey.text)

    driver.quit()

File: Selenium/Activities/Activity4.java
package activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity4 {

	public static void main(String[] args) {
		WebDriver driver = new FirefoxDriver();
		driver.get("https://v1.training-support.net/selenium/target-practice");

		System.out.println("The Title Of the Page is  : " + driver.getTitle());

		String thrirdHeader = driver.findElement(By.xpath("//h3")).getText();
		System.out.println("Third Header IS : " + thrirdHeader);

		String fifthHeaderColor = driver.findElement(By.xpath("//h5")).getCssValue("color");
		System.out.println("The clolor of the fifth header is :" + fifthHeaderColor);

		String vilotClass = driver.findElement(By.xpath("//button[text()='Violet']")).getAttribute("class");
		System.out.println("The class of Violet is :" + vilotClass);

		String greyText = driver.findElement(By.xpath("//button[text()='Grey']")).getText();
		System.out.println(greyText);

		driver.quit();
	}

}
